backend/README.md

```diff
@@ -51,7 +51,7 @@ Use `cargo clippy --all-targets --all-features -- -D warnings` to lint the proje
 
 ### Testing
 
-Use `cargo test` to run the tests. The tests are also run in a GitHub Actions workflow.
+Use `cargo test --features embed-typst` to run the tests. The tests are also run in a GitHub Actions workflow.
```

Comment:
Using the `embed-typst` feature shouldn't be needed as long as you have [Typst](https://typst.app/) installed on your machine. It's faster that way because you can avoid compiling Typst as part of Abacus.